docs/source/cluster_management/sweep/targeted-sweep.rst

```diff
@@ -104,7 +115,8 @@ The process here needs to account for the existence of old entries written to th
      This is safe; see step 2 above.
   2. Wait until targeted sweep for strategy THOROUGH has caught up to after the upgrade. This can be verified by
      consulting the ``millisSinceLastSweptTs`` targeted sweep metric.
-  3. Roll service nodes from a version with THOROUGH_MIGRATION sweep strategy to one with CONSERVATIVE.
+  3. Roll service nodes from a version with THOROUGH_MIGRATION sweep strategy to one with CONSERVATIVE. If desired,
+     the CONSERVATIVE product version may immediately begin using read-only transactions.
```

Comment:
This is fine because CONSERVATIVE is more permissive than THOROUGH(_MIGRATION) in the sense that it allows both types, whereas the others only allow one type.

Reply:
yep!